### PR TITLE
refactor increase block function

### PR DESCRIPTION
**Pull request by ducquangkstn**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -39,7 +39,7 @@
   "devDependencies": {
     "@nomiclabs/buidler": "1.1.2",
     "@nomiclabs/buidler-web3": "1.1.2",
-    "@openzeppelin/test-helpers": "0.5.4",
+    "@openzeppelin/test-helpers": "^0.5.5",
```
Comment (ilanDoron) [inline package.json]: please set as fixed version.
remove ^

So we have control when we change our work environment.

File: test/sol5/kyberDao.js [hunk 2]
```diff
@@ -146,17 +146,15 @@ contract('KyberDAO', function(accounts) {
         await deployContracts(20, currentBlock + 20, 8);
         await setupSimpleStakingData();
 
-        currentBlock = await Helper.getCurrentBlock();
-        await Helper.increaseBlockNumberBySendingEther(accounts[0], accounts[0], startBlock - currentBlock);
+        await time.advanceBlockTo(startBlock);
```
Comment (ilanDoron) [inline test/sol5/kyberDao.js]: can;t we call the helper function with this parameter?

File: test/sol5/kyberDao.js [hunk 3]
```diff
@@ -9,7 +9,7 @@ const Helper = require("../helper.js");
 const BN = web3.utils.BN;
 
 const { precisionUnits, zeroAddress } = require("../helper.js");
-const { expectEvent, expectRevert } = require('@openzeppelin/test-helpers');
+const { expectEvent, expectRevert, time } = require('@openzeppelin/test-helpers');
```
Comment (ilanDoron) [inline test/sol5/kyberDao.js]: nice if time only in helper
so if later we have issues, easy to change it back.
so all block increase or advance to can be done using our helper.

now its inconsistent.

File: package.json (hunk 1 above)
Comment (ducquangkstn) [reply package.json]: change as request

File: test/sol5/kyberDao.js (hunk 2 above)
Comment (ducquangkstn) [reply test/sol5/kyberDao.js]: added a function named `increaseBlockNumberTo` which calls `time.advanceBlockTo`

File: test/sol5/kyberDao.js (hunk 3 above)
Comment (ducquangkstn) [reply test/sol5/kyberDao.js]: change as request

**Pull request merged**